src/bp/migrations/v12_1_0-1564421788-database_srv_logs_table_has_date_timestamp.ts

```diff
@@ -0,0 +1,41 @@
+import * as sdk from 'botpress/sdk'
+import { ConfigProvider } from 'core/config/config-loader'
+import Database from 'core/database'
+import { Migration } from 'core/services/migration'
+import { Container } from 'inversify'
+
+const TABLE_NAME = 'srv_logs'
+const COLUMN_NAME = 'timestamp'
+const CURRENT_DATE_FORMAT = 'YYYY-MM-DD"T"HH24:MI:SS.MS"Z"'
+
+const migration: Migration = {
+  info: {
+    description: '',
+    type: 'config'
+  },
+  up: async (
+    bp: typeof sdk,
+    configProvider: ConfigProvider,
+    database: Database,
+    inversify: Container
+  ): Promise<sdk.MigrationResult> => {
+    const { client } = database.knex.client.config
+    if (client === 'sqlite3') {
+      return { success: true, message: 'No migration to run for sqlite' }
```

Comment:
No migration for sqlite ? So the field stays a string like before? We should be consistent with both databases, if we change to the date type and one isn't, are date parsing gonna work through knex ?

Reply:
The data doesn't need to be change, just the column type. However sqlite does'nt support alter table.

I've implemented a new solution all in knex:

1. rename timestamp to timestamp_temp
2. create new column timestamp of type timestamp
3. copy all the content of the first VARCHAR column to this one (type TIMESTAMP)

Does this seams good?

Reply:
Yep, that makes sense !